src/securityinsight/README.md

```diff
@@ -0,0 +1,184 @@
+##### List #####
+```
+az sentinel alert-rule-template list --resource-group "myRg" --workspace-name "myWorkspace"
+```
+##### Show #####
+```
+az sentinel alert-rule-template show --alert-rule-template-id "65360bb0-8986-4ade-a89d-af3cf44d28aa" \
+    --resource-group "myRg" --workspace-name "myWorkspace" 
+```
+#### sentinel bookmark ####
+##### Create #####
+```
+az sentinel bookmark create --etag "\\"0300bf09-0000-0000-0000-5c37296e0000\\"" --created "2019-01-01T13:15:30Z" \
+    --display-name "My bookmark" --labels "Tag1" --labels "Tag2" --notes "Found a suspicious activity" \
+    --query "SecurityEvent | where TimeGenerated > ago(1d) and TimeGenerated < ago(2d)" \
+    --query-result "Security Event query result" --updated "2019-01-01T13:15:30Z" \
+    --bookmark-id "73e01a99-5cd7-4139-a149-9f2736ff2ab5" --resource-group "myRg" --workspace-name "myWorkspace" 
+```
+##### Show #####
+```
+az sentinel bookmark show --bookmark-id "73e01a99-5cd7-4139-a149-9f2736ff2ab5" --resource-group "myRg" \
+    --workspace-name "myWorkspace" 
+```
+##### List #####
+```
+az sentinel bookmark list --resource-group "myRg" --workspace-name "myWorkspace"
+```
+##### Delete #####
+```
+az sentinel bookmark delete --bookmark-id "73e01a99-5cd7-4139-a149-9f2736ff2ab5" --resource-group "myRg" \
+    --workspace-name "myWorkspace" 
+```
+#### sentinel data-connector ####
+##### Create #####
+```
+az sentinel data-connector create \
+    --office-data-connector etag="\\"0300bf09-0000-0000-0000-5c37296e0000\\"" tenant-id="2070ecc9-b4d5-4ae4-adaa-936fa1954fa8" \
+    --data-connector-id "73e01a99-5cd7-4139-a149-9f2736ff2ab5" --resource-group "myRg" --workspace-name "myWorkspace" 
+```
+##### Show #####
+```
+az sentinel data-connector show --data-connector-id "763f9fa1-c2d3-4fa2-93e9-bccd4899aa12" --resource-group "myRg" \
+    --workspace-name "myWorkspace" 
+```
+##### Show #####
+```
+az sentinel data-connector show --data-connector-id "b96d014d-b5c2-4a01-9aba-a8058f629d42" --resource-group "myRg" \
+    --workspace-name "myWorkspace" 
+```
+##### Show #####
+```
+az sentinel data-connector show --data-connector-id "06b3ccb8-1384-4bcc-aec7-852f6d57161b" --resource-group "myRg" \
+    --workspace-name "myWorkspace" 
+```
+##### Show #####
+```
+az sentinel data-connector show --data-connector-id "c345bf40-8509-4ed2-b947-50cb773aaf04" --resource-group "myRg" \
+    --workspace-name "myWorkspace" 
+```
+##### Show #####
+```
+az sentinel data-connector show --data-connector-id "f0cd27d2-5f03-4c06-ba31-d2dc82dcb51d" --resource-group "myRg" \
+    --workspace-name "myWorkspace" 
+```
+##### Show #####
+```
+az sentinel data-connector show --data-connector-id "07e42cb3-e658-4e90-801c-efa0f29d3d44" --resource-group "myRg" \
+    --workspace-name "myWorkspace" 
+```
+##### Show #####
+```
+az sentinel data-connector show --data-connector-id "c345bf40-8509-4ed2-b947-50cb773aaf04" --resource-group "myRg" \
+    --workspace-name "myWorkspace" 
+```
+##### Show #####
+```
```

Comment:
is this generated? lots of show examples same same syntax

Reply:
> is this generated? lots of show examples same same syntax

Yes it's auto generated, I will refine some.